src/main/kotlin/racing/RacingCar.kt

```diff
@@ -1,18 +1,18 @@
 package racing
 
-class RacingCar(private val round: Int) {
+class RacingCar(private val round: Int, val name: String) {
```

Comment:
`round`는 프로퍼티로 안가지고 있어도 될 것 같아요!

```suggestion
class RacingCar(round: Int, val name: String) {
```

Reply:
[class layout](https://kotlinlang.org/docs/coding-conventions.html#class-layout)에 맞게 companion object와 프로퍼티들의 순서를 재정렬해보면 어떨까요?